### DISCUSSION TITLE
Welcome to ComfyUI_frontend Discussions!

### DISCUSSION THREAD

**Discussion by huchenlei** [Comfy-Org/ComfyUI_frontend#105] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dicksondickson** [comment]
What are some of the new things planned for the frontend? Can't help but feel amazing things are happening.

**Reply by huchenlei** [reply]
The roadmap in README has some items we thought are valuable a while ago (Last month). If you have any ideas, definitelly share it here and I can add them to the roadmap
